apps/server/doc/architecture-mapping-legacy.md

```diff
@@ -1,219 +0,0 @@
-# Architecture mapping to Code
-
-## Conventions
```

Comment:
Do we have any other place where these conventions are documented?

Reply:
I added that file back